### PR TITLE
Fix misleading documentation for naming blocks

### PR DESCRIPTION
**Pull request by FrontSide**
##### SUMMARY

From what I have observed it is the block itself that doesn't support the name attribute rather than the tasks inside the block.
The error I'm getting when I try to use a named block with Ansible 2.0 is: `'name' is not a valid attribute for a Block`

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
blocks documentation

##### ADDITIONAL INFORMATION

According to the current documentation the following should fail to execute in Ansible versions prior to 2.3:

```
- block:

       - name: Task 1
          debug: 
              msg: "Some Task"

       - name: Task 2
          debug: 
              msg: "Some other Task"
```

This is however not the case and it executes fine. 
The following however fails due to the name attribute in the block.

```
- name: "Block that executes some tasks" 
   block:

       - name: Task 1
          debug: 
              msg: "Some Task"

       - name: Task 2
          debug: 
              msg: "Some other Task"
```
```
ERROR! 'name' is not a valid attribute for a Block
```

### CODE REVIEW

File: docs/docsite/rst/user_guide/playbooks_blocks.rst
Comment (acozine) [inline docs/docsite/rst/user_guide/playbooks_blocks.rst]:
```suggestion
Names for blocks have been available since Ansible 2.3. We recommend using names in all tasks, within blocks or elsewhere, for better visibility into the tasks being executed when you run the playbook.
```

**Review by acozine — DISMISSED** [review]
Thanks @FrontSide for your interest in improving the Ansible documentation.

Since Ansible 2.0 and Ansible 2.3 are both unmaintained and may contain unfixed security vulnerabilities, we strongly recommend you upgrade to a maintained version. For this same reason, we are moving away from calling out differences among unmaintained versions of Ansible.

If you're willing to accept my suggested wording, I'm willing to merge the PR, though.

Please note that Ansible is also going through a major overhaul in preparation for the release of 2.10. We are holding off on publishing updates to the docsite until we have replaced the old module documentation with documentation from the collections proposed for Ansible Community Distribution, so even once the change is merged, it may not show up on the docsite for some time.

File: docs/docsite/rst/user_guide/playbooks_blocks.rst
Comment (FrontSide) [reply docs/docsite/rst/user_guide/playbooks_blocks.rst]: Hey @acozine. Thanks for the quick review.

As for the suggested rewording. 
It says `Names for blocks themselves and for tasks within blocks have been available since 2.3`. 

Whenever I try to run a block that has tasks with names with an Ansible version prior 2.3 it seems to run fine, which is why this seems confusing to me, as I would expect it to fail.
For me it is only when I add a name to the block itself that it errors.

I understand that we are talking about very old versions here, I would just like to avoid publishing wrong information.

Thanks

Comment (acozine) [reply docs/docsite/rst/user_guide/playbooks_blocks.rst]: @FrontSide I revised the suggestion.

**Comment by acozine** [comment]
@FrontSide I went ahead and committed my suggestion. Our publication pipeline doesn't print out every `version_added` tag - it [ignores them for older versions](https://github.com/ansible/ansible/blob/stable-2.9/hacking/build_library/build_ansible/command_plugins/plugin_formatter.py#L57) - so the information you added would not change the final HTML.

**Pull request merged**
